bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/InsteonBinding.java

```diff
@@ -0,0 +1,512 @@
+ * path because it had led to a complicated and fragile system which due to the technical limitations
+ * above was inherently squirrely.
+ *
+ *
+ * @author Bernd Pfrommer - Initial contribution
+ * @author Daniel Pfrommer - OpenHAB 1 insteonplm binding
+ * @author Rob Nielsen - Port to OpenHAB 2 insteon binding
+ */
+@NonNullByDefault
+public class InsteonBinding {
+    private static final int DEAD_DEVICE_COUNT = 10;
+
+    private final Logger logger = LoggerFactory.getLogger(InsteonBinding.class);
+
+    private Driver m_driver = new Driver();
+    private ConcurrentHashMap<InsteonAddress, InsteonDevice> m_devices = new ConcurrentHashMap<InsteonAddress, InsteonDevice>();
+    private ConcurrentHashMap<String, InsteonChannelConfiguration> m_bindingConfigs = new ConcurrentHashMap<>();
+    private PortListener m_portListener = new PortListener();
+    private int m_devicePollIntervalMilliseconds = 300000;
+    private int m_deadDeviceTimeout = -1;
+    private int m_messagesReceived = 0;
+    private boolean m_isActive = false; // state of binding
+    private int m_x10HouseUnit = -1;
+    private InsteonNetworkHandler m_handler;
+
+    public InsteonBinding(InsteonNetworkHandler handler, @Nullable InsteonNetworkConfiguration config) {
+        this.m_handler = handler;
+
+        Integer devicePollIntervalSeconds = config.getDevicePollIntervalSeconds();
+        if (devicePollIntervalSeconds != null) {
+            m_devicePollIntervalMilliseconds = devicePollIntervalSeconds * 1000;
+        }
+        logger.info("device poll interval set to {} seconds", m_devicePollIntervalMilliseconds / 1000);
+
+        Integer modemDbRetryTimeoutSeconds = config.getModemDbRetryTimeoutSeconds();
+        if (modemDbRetryTimeoutSeconds != null) {
+            logger.info("setting modem db retry timeout to {} seconds", modemDbRetryTimeoutSeconds);
+            m_driver.setModemDBRetryTimeout(modemDbRetryTimeoutSeconds * 1000);
+        }
+
+        String additionalDevices = config.getAdditionalDevices();
+        if (additionalDevices != null) {
+            try {
+                DeviceTypeLoader.s_instance().loadDeviceTypesXML(additionalDevices);
+                logger.info("read additional device definitions from {}", additionalDevices);
+            } catch (ParserConfigurationException | SAXException | IOException e) {
+                logger.warn("error reading additional devices from {}", additionalDevices, e);
+            }
+        }
+
+        String additionalFeatures = config.getAdditionalFeatures();
+        if (additionalFeatures != null) {
+            logger.info("reading additional feature templates from {}", additionalFeatures);
+            DeviceFeature.s_readFeatureTemplates(additionalFeatures);
+        }
+
+        m_deadDeviceTimeout = m_devicePollIntervalMilliseconds * DEAD_DEVICE_COUNT;
+        logger.info("dead device timeout set to {} seconds", m_deadDeviceTimeout / 1000);
```

Comment:
```suggestion
        logger.debug("dead device timeout set to {} seconds", m_deadDeviceTimeout / 1000);
```

Reply:
This is a useful message to print out at startup, and has always been this way.

Reply:
I'm going to leave as is since it's only logged during initialization and contains useful info.

Reply:
done